.github/workflows/check_format.yml

```diff
@@ -11,7 +11,7 @@
 name: Check format
 
 on:
-  - pull_request
+  - push
```

Comment:
This file still needs its changes removed.

Reply:
Did you forget to push? This is not yet resolved.

Reply:
I pushed yesterday with this review.